rktl_perception/launch/focus_assist.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
```

Comment:
This should be modified to start each camera. That way we don't have to run the full stack just to focus the cameras.

Reply:
Good point.